### PR TITLE
Initialize tutor app

### PR DESCRIPTION
**Pull request by mosesnagar**
#6 :

Install flask and postgresSQL.
Handle requirements file to install pip packages. Install SQLAlchemy.
Configure models.py file in order to maintain models.

Create hello world route, with "/" route.
Create route to get all the models, with "/models".
Create route to add new model, with "/model/store".

### CODE REVIEW

**Review by ifireball — CHANGES_REQUESTED** [review]
I mentioned some issues I see with what seems to be the design. Also I recomment splitting this big PR into different PRs - installing the db, installing ORM, setting up the models.

If it was split - we could for example merge the DB installation already since it generally looks ok.

File: tutor/controllers/model.py
Comment (mosesnagar) [reply tutor/controllers/model.py:6]: It was an example, but you are right - it can be more confusing. I will change it to functions.

File: tutor/models.py
Comment (mosesnagar) [reply tutor/models.py:4]: Yes, it is an example - not a real model in our app.

File: tutor/routing.py
Comment (mosesnagar) [reply tutor/routing.py]: I added a routes folder, so every feature may have is own route file. So every teammate can handle its own files.

File: bootstrap.sh
Comment (golan13) [inline bootstrap.sh:5]: I suggest also upgrading the VM when spinning up the environment.
apt update && sudo apt upgrade -y

Comment (ifireball) [reply bootstrap.sh]: Sorry, that is bad advice - it would make your development environment not reproducible. Usually you should control versions tightly and not allow "blanket upgrades" like this.

**Comment by ifireball** [comment]
I'm not seeing this PR being split apart as I've asked. Do you intend to do this?

File: bootstrap.sh
Comment (golan13) [reply bootstrap.sh]: good point. thanks

**Comment by mosesnagar** [comment]
> I'm not seeing this PR being split apart as I've asked. Do you intend to do this?

I thought it was a comment for future PRs, because that on that one I already fixed the issues. Do you want me to split that PR? 
As you have enlightened, we will look at python venv and to include it in our app(on a different PR).

**Comment by ifireball** [comment]
Yes, I haven't looked at the code for this PR seriously yet because its so big.

**Comment by mosesnagar** [comment]
I created new Prs. #14  #15  #16 - that their order. They all based on each other, so I couldn't just create a new one without the other, but I think after the first approval the unchanged files will removed from the "files changed" section on the PR so it wouldn't be a lot of files for every PR.

**Comment by ifireball** [comment]
Do you intend to just close this one?

**Pull request closed without merging**
